tensorflow/compiler/xla/pjrt/gpu_device.cc

```diff
@@ -166,13 +170,19 @@ class GpuClient : public xla::PjRtStreamExecutorClient {
       int num_replicas, int num_partitions) const override;
 
   absl::string_view platform_version() const override {
-#if defined(CUDART_VERSION)
+#if defined(TF_ROCM_VERSION) || defined(CUDART_VERSION)
```

Comment:
Can we make this conditional on `TENSORFLOW_USE_ROCM` only?

Reply:
Relying only TENSORFLOW_USE_ROCM is not a good idea, as we expect this from this function to attach a version number to "rocm "